src/ukify/ukify.py

```diff
@@ -25,24 +25,30 @@
 
 import argparse
 import configparser
+import contextlib
 import collections
 import dataclasses
+import datetime
 import fnmatch
 import itertools
 import json
 import os
 import pathlib
 import pprint
+import pydoc
 import re
 import shlex
 import shutil
+import socket
 import subprocess
 import sys
 import tempfile
 from typing import (Any,
                     Callable,
                     IO,
                     Optional,
+                    Sequence,
+                    Tuple,
```

Comment:
Starting with 3.9 you can use `tuple` for typing where `Tuple` would have been necessary before.

Reply:
I always get lost between all the different versions we support in different places. But indeed, ukify has an explicit check for 3.9, so `tuple` can be used.